antipatterns/silver_bullet.json

```diff
@@ -0,0 +1,25 @@
+{
+  "antiPatterns": [
+    {
+      "name": "Silver Bullet",
+      "aliases": [
+        "Web Services Will Fix Our Problems"
+      ],
+      "description": "Describes a number of web services that are used to solve a problem on top of an existing architecture. This comes from the assumption that (web-)services will fix any problem. If the underlying architecture has lacks in his structure, e.g., a bloated session, then the webservice will exhibit the this structure with a bloated web service. ",
+      "context": "",
+      "detection": "Typically have a number of web services on top of an existing architecture.",
+      "consequences": "There is either a significant refactoring of the logic or if just copy and pasted there is a maintenance overhead. Also the web services that are created in this way are hard to reuse, because they are for this specific purpose and tightly coupled. Additionally, there is a development overhead for the web services.",
```

Comment:
```suggestion
      "consequences": "There is either a significant refactoring of the logic or if just copy and pasted there is a maintenance overhead. Also the web services that are created in this way are hard to reuse, because they were built for this specific purpose and tightly coupled. Additionally, there is a development overhead for the web services.",
```